meson.build

```diff
@@ -1,33 +1,39 @@
-# Define project name
```

Comment:
I would like to keep these comments seen throughout the meson builds. Helps me in future projects (because I do reuse the meson builds and modify them to my needs), and may help other people know what's going on. Granted, a lot of the meson functions are pretty self explanatory

Reply:
I think, as you said, that the meson functions and variable names are explicit enough for any developer to understand this.